internal/providers/terraform/google/compute_per_instance_config.go

```diff
@@ -0,0 +1,25 @@
+package google
+
+import (
+	"github.com/infracost/infracost/internal/resources/google"
+	"github.com/infracost/infracost/internal/schema"
+)
+
+func getComputePerInstanceConfigRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:                "google_compute_per_instance_config",
+		RFunc:               newComputePerInstanceConfig,
+		ReferenceAttributes: []string{"instance_group_manager"},
+	}
+}
+
+func newComputePerInstanceConfig(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+
+	r := &google.ComputePerInstanceConfig{
+		Address: d.Address,
+	}
+
+	r.PopulateUsage(u)
+
+	return r.BuildResource()
```

Comment:
Since this resource has no cost component you can make it free like this: 
```suggestion
	return &schema.RegistryItem{
		Name:                "google_compute_per_instance_config",
		NoPrice: true,
		ReferenceAttributes: []string{"instance_group_manager"},
		Notes: []string{"Free resource."},
	}
```

Once that is done you can also remove internal/resources/google/compute_per_instance_config.go